mods/ctf_pvp_engine/ctf_chat/init.lua

```diff
@@ -24,10 +24,20 @@ local function team_console_help(name)
 	end
 end
 
+local function access_console_help(name)
+	minetest.chat_send_player(name, "Try:")
+	minetest.chat_send_player(name, "/access add <all/flag> <team/player> - Allows either a whole team or player to interact with your protection")
+	minetest.chat_send_player(name, "/access remove <all/flag> <team/player> - Removes a team or player from your protection.")
+	minetest.chat_send_player(name, "/access clear <all/flag> <all/teams/players> - Clears players, teams ,or all from your protection")
+	minetest.chat_send_player(name, "/access list <all/flag> - List who has access to your protection")
+	minetest.chat_send_player(name, "/access open <all/flag> - allow all or the inputed flag access to all members")
+	minetest.chat_send_player(name, "/access close <all/flag> - Make all or the inputed flag white list only")
+end
```

Comment:
Multiple `chat_send_player` calls isn't a great idea. Use multi-line string literals syntax (`[[` and `]]`) like so:

```lua
minetest.chat_send_player(name, [[Access help:
	/access add ...
	/access remove ...
	...
	...
	/access close
]])
```